statblock-converter/scripts/sbcParsers.js

```diff
@@ -2311,8 +2311,8 @@ class attacksParser extends sbcParserBase {
                     }
 
                     // attackName
-                    if (attack.match(/(\b[a-zA-Z*]+)(?:[ +0-9(/]+\(*)/) !== null) {
-                        attackName = attack.match(/(\b[a-zA-Z *]+)(?:[ +0-9(/]+\(*)/)[1].replace(/^ | $/g, "").replace(/\bmwk\b /i, "").replace(/\*/, "")
+                    if (attack.match(/((?:[a-zA-Z]| (?=[a-zA-Z])|\*)+)(?:[ +0-9(/]+\(*)/) !== null) {
+                        attackName = attack.match(/((?:[a-zA-Z]| (?=[a-zA-Z])|\*)+)(?:[ +0-9(/]+\(*)/)[1].replace(/^ | $/g, "").replace(/\bmwk\b /i, "").replace(/\*/, "").trim()
```

Comment:
This matches "d" for attacks without a given name, like `Melee +8 (1d10+6 plus disease and trip)`.
Should not be an issue, if that's not something happening in the statblocks, but i'll keep a look out for this.